live-examples/js-examples/dataview/dataview-getbigint64.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">// create an ArrayBuffer with a size in bytes
+var buffer = new ArrayBuffer(16);
```

Comment:
I know this is amazingly pedantic even for me but I would like a blank line between this one and the next.

Reply:
Looks cleaner that way indeed. Updated!